BDArmory/Ammo/PooledBullet.cs

```diff
@@ -1420,136 +1420,143 @@ public void BeehiveDetonation()
                 Debug.Log("[BDArmory.PooledBullet] Beehive round not configured with subMunitionType!");
                 return;
             }
-            string fuze = subMunitionType.fuzeType.ToLower();
-
-            BulletFuzeTypes sFuze;
-            switch (fuze)
+            string[] subMunitionData = subMunitionType.Split(new char[] { ';' });
+            string projType = subMunitionData[0];
+            int count = int.Parse(subMunitionData[1]);
```

Comment:
Actually, this would be better with
```csharp
if (subMunitionData.Length < 2 || !int.TryParse(subMunitionData[1], out int count)) count = 1;
```
and the extra parse on lines 1003—1005 of PooledRocket.cs isn't needed since it's already parsed on lines 866—868.